app/controllers/api/base_controller/parameters.rb

```diff
@@ -82,6 +82,9 @@ def parse_filter(filter, operators)
                         [filter_value.gsub(/^'|'$/, ''), str_method]
                       when /^".*"$/
                         [filter_value.gsub(/^"|"$/, ''), str_method]
+                      when /^\d{4}-\d{2}-\d{2}/
```

Comment:
We need to discuss what the API support if just yyyy-mm-dd, or more, i.e. Using Date.iso8601(...) to parse or a subset of it.   What does MiqExpression support ?